src/oci/registry.rs

```diff
@@ -0,0 +1,21 @@
+use crate::oci::image::manifest;
+use crate::oci::image::manifest::Manifest;
+use std::error;
+
+pub fn get(path: String) -> Result<Manifest, Box<dyn error::Error>> {
+    let res = ureq::get("http://httpbin.org/get").call().into_string()?;
+
+    manifest::from_str(&res)
+}
+
+#[test]
+fn get_ok() {
+    let r = get("localhost:5000/test:local".to_string());
+
+    if r.is_err() {
```

Comment:
Why not using assert as in previous tests?

Reply:
debugging purpose ;)